app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -285,7 +285,7 @@ export default Component.extend(ComposerUploadUppy, {
         count: minimumPostLength,
       });
       const tl = this.get("currentUser.trust_level");
-      if (tl === 0 || tl === 1) {
+      if ((tl === 0 || tl === 1) && !this.get("_isNewTopic")) {
```

Comment:
It can be shortened to:

```suggestion
      if ((tl === 0 || tl === 1) && !this._isNewTopic) {
```